README.local-opsworks.md

```diff
@@ -0,0 +1,96 @@
+# Local opsworks development
+
+This feature allows you to spin up clusters that match our deployed opsworks
+clusters very closely in a local vagrant environment.
+
+## Requirements
+
+* virtualbox 5.0.x
+* vagrant 1.8.x
+* [vagrant-hosts plugin](https://github.com/oscar-stack/vagrant-hosts)
+* mh-opsworks-recipes and matterhorn-dce-fork must be checked out as sibling
+  directories of this repository.
+
+## Getting started
+
+* Checkout mh-opsworks and run `./bin/setup`.
+* Install the required versions of vagrant and virtual box
+* Install the vagrant-hosts plugin:
+
+        vagrant plugin install vagrant-hosts --plugin-version 2.6.2
+
+* Ensure your repository directory structure looks like this:
+
+        [parent dir]
+          - mh-opsworks
+          - mh-opsworks-recipes
+          - matterhorn-dce-fork
+
+* Start your environment! This takes about 20 to 25 minutes on my beefy machine
+  with SSD.
+
+        ./bin/local_cluster up
+
+* Access your environment by pointing your web browser to http://10.10.10.10/
+  The default password is "fakepass".
+
+## `./bin/local_cluster`
+
+This harness helps you spin up a multi-machine cluster. Run it without
+arguments to see what it supports. It ultimately use the `vagrant` command
+under the covers for most actions, but parallelizes most processes for
+speed.
+
+## More info
+
+By default we set up 5 nodes:
+
+* storage1 - the nfs share
+* db-master1 - the database server
+* admin1
+* engage1
+* workers1
+
+* Access your actual instances via the default vagrant tooling:
+
+        # List all running vagrant machines
+        vagrant global-status
+        # get into a specific node. You can sudo to do stuff as root.
+        vagrant ssh admin
+
+* Your local matterhorn-dce-fork and chef recipe repos are available in the
+  vagrant machines at:
+
+        /vagrant/matterhorn-dce-fork
+        /vagrant/mh-opsworks-recipes
+
+* Do work locally. After you've done stuff, redeploy your changes. This takes
+  around 5 minutes (for now).
+
+        ./bin/local_cluster deploy
+
+* Stop your cluster
+
+        ./bin/local_cluster halt
```

Comment:
Dan, If I halt my cluster, is that the same as vagrant suspend? How do I restart and existing halted cluster?